srcs/back/src/friends/friends.controler.ts

```diff
@@ -0,0 +1,33 @@
+import { Body, Controller, Get, Post, UseGuards } from "@nestjs/common";
+import { AuthGuard } from "@nestjs/passport";
+import { FriendsService } from "./friends.service";
+import { JwtPayload } from "src/auth/jwt/jwt-auth.strategy";
+import { UserDecorator } from "src/users/decorator/users.decorator";
+
+@UseGuards(AuthGuard)
```

Comment:
```suggestion
@UseGuards(AuthGuard)
```
il me semble qu'il faut utiliser le jwt2fa guard ici sinon ca pourrait poser probleme (voir le user controller)